specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/preview/2016-01-01/DownloadedProduct.json

```diff
@@ -154,6 +154,53 @@
         "consumes": [
           "application/json"
         ]
+      },
+      "put": {
+        "x-ms-examples": {
+          "Puts the specified downloaded product.": {
+            "$ref": "./examples/DownloadedProduct/Put.json"
+          }
+        },
+        "description": "Creates a downloaded product.",
+        "tags": [
+          "DownloadedProducts"
+        ],
+        "operationId": "DownloadedProducts_Create",
+        "parameters": [
+          {
+            "$ref": "AzureBridge.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "AzureBridge.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "AzureBridge.json#/parameters/ActivationNameParameter"
+          },
+          {
+            "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+          },
+          {
+            "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DownloadedProductPutParameter"
```

Comment:
Can you share a common type definition with Get? DownloadedProductResource

ARM resources should be put-able with the response from the get request.

Reply:
Updated the API